blocks/init/src/Blocks/components/jumbotron/jumbotron-style.scss

```diff
@@ -32,10 +32,10 @@
 	}
 
 	&__heading {
-		margin-bottom: 1em;
+		padding-bottom: 1em;
```

Comment:
Was there something wrong with margins? 😄

Reply:
yes to override browser default we used margin: 0 on paragraph and heading, and because jumotron is after heading in the alphabet heading style overrides jumbotron :D

Reply:
That could be fun to override on future projects where padding might not always be an options, but let's leave it like thi s for now 😄